test/azure/version-tolerant/AcceptanceTests/asynctests/test_paging.py

```diff
@@ -53,178 +53,184 @@ async def custom_url_client():
+    assert len(items) == 10
+    assert items[-1]["properties"]["id"] == 110
+
+@pytest.mark.asyncio
+async def test_get_single_pages_failure(client):
+    pages = client.paging.get_single_pages_failure()
+    with pytest.raises(HttpResponseError):
+        async for i in pages:
+            ...
+
+@pytest.mark.asyncio
+async def test_get_multiple_pages_failure(client):
+    pages = client.paging.get_multiple_pages_failure()
+    with pytest.raises(HttpResponseError):
+        async for i in pages:
+            ...
+
+@pytest.mark.asyncio
+async def test_get_multiple_pages_failure_uri(client):
+    pages = client.paging.get_multiple_pages_failure_uri()
+    with pytest.raises(HttpResponseError):
+        async for i in pages:
+            ...
+
+@pytest.mark.asyncio
+async def test_paging_fragment_path(client):
+
+    pages = client.paging.get_multiple_pages_fragment_next_link(api_version="1.6", tenant="test_user")
+    items = []
+    async for item in pages:
+        items.append(item)
+    assert len(items) == 10
+
+    with pytest.raises(AttributeError):
+        # Be sure this method is not generated (Transform work)
+        await client.paging.get_multiple_pages_fragment_next_link_next()  # pylint: disable=E1101
+
+@pytest.mark.asyncio
+async def test_custom_url_get_pages_partial_url(custom_url_client):
+    pages = custom_url_client.paging.get_pages_partial_url("local")
+    items = []
+    async for item in pages:
+        items.append(item)
+
+    assert len(items) == 2
+    assert items[0]["properties"]["id"] == 1
+    assert items[1]["properties"]["id"] == 2
+
+@pytest.mark.asyncio
+async def test_custom_url_get_pages_partial_url_operation(custom_url_client):
+    pages = custom_url_client.paging.get_pages_partial_url_operation("local")
+    items = []
+    async for item in pages:
+        items.append(item)
+
+    assert len(items) == 2
+    assert items[0]["properties"]["id"] == 1
+    assert items[1]["properties"]["id"] == 2
+
+@pytest.mark.asyncio
+async def test_get_multiple_pages_lro(client):
+    """LRO + Paging at the same time.
+    """
+    from azure.mgmt.core.polling.async_arm_polling import AsyncARMPolling
+    poller = await client.paging.begin_get_multiple_pages_lro(polling=AsyncARMPolling(timeout=0))
+    pager = await poller.result()
+    items = []
+    async for item in pager:
+        items.append(item)
+
+    assert len(items) == 10
+    assert items[0]["properties"]["id"] == 1
+    assert items[1]["properties"]["id"] == 2
+
+@pytest.mark.asyncio
+async def test_initial_response_no_items(client):
+    pages = client.paging.first_response_empty()
+    items = []
+    async for item in pages:
+        items.append(item)
+    assert len(items) == 1
+
+@pytest.mark.asyncio
+async def test_item_name_with_xms_client_name(client):
+    pages = client.paging.get_paging_model_with_item_name_with_xms_client_name()
+    items = []
+    async for item in pages:
+        items.append(item)
+    assert len(items) == 1
+
```

Comment:
ignore all the diff except the addition of the last test. Wanted to get rid of the unnecessary test class for the async paging test